src/components/Onboarding.jsx

```diff
@@ -7,144 +7,106 @@ export default function Onboarding() {
         <div className="d-flex justify-content-center mb-2">
-          <rb.Button size="lg" className="col-10" variant="dark" onClick={() => setStep(1)}>
+          <rb.Button size="lg" variant="dark" onClick={next}>
             Get started
           </rb.Button>
         </div>
         <div className="d-flex justify-content-center">
-          <rb.Button
-            size="lg"
-            className="col-10"
-            variant="outline-dark"
-            onClick={() => settingsDispatch({ showOnboarding: false })}
-          >
+          <rb.Button size="lg" variant="outline-dark" onClick={() => settingsDispatch({ showOnboarding: false })}>
             Skip intro
           </rb.Button>
         </div>
-        <p className="mt-4 text-center">
+        <p className="mt-4 text-center text-secondary">
           Your wallet, your coins. <br />
           100% open-source & open-design.
         </p>
-      </div>
-    )
-  } else if (step === 1) {
-    return (
-      <div className="mx-auto my-auto py-5 col-10 col-md-4">
-        <div className="d-flex justify-content-center">
-          <Sprite symbol="welcome" width="150" height="150" className="mb-4" />
-        </div>
-        <p className="fs-3 text-center">Welcome to JoinMarket!</p>
-        <p className="text-center text-secondary mb-5">
-          JoinMarket is a privacy-focused software solution that aims to improve the confidentiality and privacy of your
-          bitcoin transactions. It facilitates the creation of collaborative transactions through a peer-to-peer
-          marketplace. <br />
-          <br />
-          ⚠️ Warning: While JoinMarket is tried and tested, this user interface is not. If anything breaks please{' '}
-          <a
-            href="https://github.com/joinmarket-webui/joinmarket-webui/issues"
-            target="_blank"
-            rel="noopener noreferrer"
-            className="link-secondary"
-          >
-            report an issue
-          </a>{' '}
-          on GitHub.
-        </p>
-        <div className="d-flex justify-content-center">
-          <rb.Button size="lg" className="col-6" variant="dark" onClick={() => setStep(2)}>
-            Next
-          </rb.Button>
-        </div>
-      </div>
-    )
-  } else if (step === 2) {
-    return (
-      <div className="mx-auto my-auto py-5 col-10 col-md-4">
-        <div className="d-flex justify-content-center">
-          <Sprite symbol="collab" width="150" height="150" className="mb-4" />
-        </div>
-        <p className="fs-3 text-center">Collaborative Transactions</p>
-        <p className="text-center text-secondary mb-5">
-          To have strong privacy guarantees in the open and transparent world of bitcoin, special kinds of transactions
-          have to be created. JoinMarket helps you to create these transactions in an easy and automated way.
+        <p className="mt-4 text-secondary text-center">
+          <div className="fw-bolder">Warning</div>
```

Comment:
Procudes a warning in the browser console `react-dom.development.js:67 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`

Reply:
👍  Fixed!